demisto_sdk/commands/content_graph/objects/pre_process_rule.py

```diff
@@ -0,0 +1,9 @@
+from typing import Set
+
+from demisto_sdk.commands.content_graph.common import ContentType
+from demisto_sdk.commands.content_graph.objects.base_content import BaseContent
+
+
+class PreProcessRule(BaseContent, content_type=ContentType.PREPROCESS_RULE):
+    def metadata_fields(self) -> Set[str]:
+        raise NotImplementedError("PreprocessRule not included in metadata")
```

Comment:
why not? also, where will it be caught?

Reply:
The server expects `name` and `description`, so I will show those.
However, we still have an issue where our `metdata.json` is not the same `metadata.json` the Upload Flow produces which will be fixed by @yaakovpraisler